modules/sdk/src/main/scala/org/tessellation/sdk/http/p2p/clients/L0GlobalSnapshotClient.scala

```diff
@@ -59,6 +61,12 @@ object L0GlobalSnapshotClient {
         PeerResponse[F, Signed[GlobalIncrementalSnapshot]](s"global-snapshots/${ordinal.value.value}")(client)
       }
 
+      def getHash(ordinal: SnapshotOrdinal): PeerResponse[F, Option[Hash]] = {
+        import org.http4s.circe.CirceEntityCodec.circeEntityDecoder
+
+        PeerResponse[F, Option[Hash]](s"global-snapshots/hash/${ordinal.value.value}")(client)
```

Comment:
`hash` and `ordinal` are a property of snapshot. `ordinal` is not a property of `hash`, so IMO the URL should be:
`/global-snapshots` -> list of all snapshots
`/global-snapshots/{ordinal}` -> one snapshot by ordinal
`/global-snapshots/{ordinal}/hash` -> hash of that snapshot
cc: @AlexBrandes

Reply:
I will need guidance here.

```scala
    case GET -> Root / SnapshotOrdinalVar(ordinal) :? FullSnapshotQueryParam(fullSnapshot) / "hash" =>
```

gives me

```
pattern type is incompatible with expected type;
 found   : org.http4s.Uri.Path
 required: scala.collection.immutable.Map[String,scala.collection.Seq[String]]
```

Reply:
I think it's because the url you constructed `/global-snapshots/{ordinal}?full/hash` is invalid.
see https://en.wikipedia.org/wiki/Query_string and https://www.ibm.com/docs/en/rtw/9.0.0?topic=resources-creating-web-url-schema

Reply:
@cngo-github the URL looks to be in different order (`ordinal` is not a property of `hash`, so it shouldn't be `/hash/{ordinal}`). Take a look at a suggested URL above

Reply:
The route has been changed to `/global-snapshots/${ordinal}/hash`. There is no route for full snapshots.